source-code/core/src/lint/rules/missingKey.ts

```diff
@@ -0,0 +1,24 @@
+import type { Reporter } from '../reporter.js'
+import { createRule } from '../rule.js'
+
+export const missingKeyRule = createRule('inlang.missingKey', () => {
+	let reporter: Reporter
+	let referenceLanguage: string
+
+	return {
+		initialize: (config) => {
+			reporter = config.reporter
+			referenceLanguage = config.referenceLanguage
+		},
+		visitors: {
+			Resource: ({ target }) => {
+				if (target && target.languageTag.name === referenceLanguage) return 'skip'
```

Comment:
it could be a common use case to skip the processing of the reference resource. Should we add a flag to the configuration so rule authors don't need to add these 3 lines of code?

Reply:
I vote to refrain from adding sugar for now. Let's try to keep the surface area of the API as small as possible.